clients/php/predis.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "predis",
+    "description": "A flexible and feature-complete Redis client for PHP.",
+    "repo":"https://github.com/predis/predis",
+    "installation": "composer require predis/predis",
+    "language":"php",
+    "package_size": "2.8M",
+    "license": "MIT",
+    "read_from_replica": true,
+    "smart_backoff_to_prevent_connection_storm": true,
+    "pubsub_state_restoration": false,
+    "cluster_scan": false,
+    "latency_based_read_from_replica": false,
+    "AZ_based_read_from_replica": false,
+    "client_side_caching": false,
+    "client_capa_redirect": false,
+    "persistent_connection_pool": false
+}
```

Comment:
Any change you can add Relay as well?

```json
{
    "name": "Relay",
    "description": "Next-generation caching layer for Valkey/Redis.",
    "repo": "https://github.com/cachewerk/relay",
    "installation": "https://relay.so/docs/installation",
    "language": "php",
    "package_size": "2.5M",
    "license": "Proprietary",
    "read_from_replica": true,
    "smart_backoff_to_prevent_connection_storm": true,
    "pubsub_state_restoration": false,
    "cluster_scan": false,
    "latency_based_read_from_replica": false,
    "AZ_based_read_from_replica": false,
    "client_side_caching": true,
    "client_capa_redirect": false,
    "persistent_connection_pool": true
}
```

Reply:
Proprietary? I'd say no to that. We do open source only.

Reply:
The main repo seems to be using an MIT license, is it actually proprietary?